src/SelectState.js

```diff
@@ -44,6 +44,7 @@ class Select extends Component {
         identifier={this.props.identifier}
         label={this.props.label}
         options={this.state.options}
+        selected={this.state.selected || []}
```

Comment:
probably a good idea to keep these as stateless functions and keep all the states in redux store